_includes/3d-secure-2.md

```diff
@@ -0,0 +1,140 @@
+{% assign api_resource = include.api_resource %}
```

Comment:
Codacy found an issue: [[final-newline] Missing newline character at end of file](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5786122)